src/cli/codev-account.ts

```
@@ -0,0 +1,157 @@
+    case 'list':
+    case 'ls':
+      printList();
+      return 0;
+
+    case 'add': {
+      const label = rest.find((a) => !a.startsWith('-'));
+      const dir = getFlag(rest, '--dir');
+      const a = manager.addAccount(label as string, { dir });
+      console.log(`✓ Added "${a.label}"  (${a.dir})`);
+      if (!a.loggedIn) {
+        console.log(`  Log in with:  claude ${a.label}   (or claude-${a.label})`);
```

ESLint

[error] Replace ``··Log·in·with:··claude·${a.label}···(or·claude-${a.label})`` with `⏎··········`··Log·in·with:··claude·${a.label}···(or·claude-${a.label})`,⏎········`

(prettier/prettier)